app/models/mqtt_client_task.rb

```diff
@@ -1,16 +1,16 @@
 class MqttClientTask < Task
   def dispatch
     catalog_inventory_url = ClowderConfig.instance["CATALOG_INVENTORY_EXTERNAL_URL"]
-    mqtt_client_url = ClowderConfig.instance["MQTT_CLIENT_URL"]
+    cloud_connector_url = ClowderConfig.instance["CLOUD_CONNECTOR_URL"]
     app_version = ClowderConfig.instance["CURRENT_API_VERSION"] || "v1.0"
 
     task_url_prefix = File.join(catalog_inventory_url, app_version, "tasks").gsub(/^\/+|\/+$/, "")
-    opts = {:task_id          => id,
-            :task_url         => task_url_prefix,
-            :mqtt_client_guid => source.mqtt_client_id,
-            :mqtt_client_url  => mqtt_client_url}
+    opts = {:task_id             => id,
+            :task_url            => task_url_prefix,
+            :mqtt_client_guid    => source.mqtt_client_id,
```

Comment:
@hsong-rh Do you want to rename the column name from mqtt_client_id to cloud_controller_id, we also need to fix the openapi.json